### PR TITLE
Update ADOPTERS.md for PropulsionAI

### PR DESCRIPTION
**Pull request by Rohithzr**
_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [x] Changelog has been updated and is aligned with our [changelog requirements](https://github.com/kedacore/keda/blob/main/CONTRIBUTING.md#Changelog)
- [x] Any necessary documentation is added, such as:
  - [`README.md`](/README.md)
  - [The `docs/` directory](./docs)
  - [The docs repo](https://github.com/kedacore/keda-docs)

Fixes #

### CODE REVIEW

File: ADOPTERS.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ This page contains a list of organizations who are using KEDA's HTTP Add-on in p
 | Organization | Status | More Information (Blog post, etc.) |
 | ------------ | ---------| ---------------|
 |N/A|![testing](https://img.shields.io/badge/-development%20&%20testing-green?style=flat)| N/A|
```
Comment (tomkerkhove) [inline ADOPTERS.md:9]: Mind removing this one?

File: ADOPTERS.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ This page contains a list of organizations who are using KEDA's HTTP Add-on in p
 | Organization | Status | More Information (Blog post, etc.) |
 | ------------ | ---------| ---------------|
 |N/A|![testing](https://img.shields.io/badge/-development%20&%20testing-green?style=flat)| N/A|
+|PropulsionAI|![testing](https://img.shields.io/badge/-development%20&%20testing-green?style=flat)|Backend API and Frontend Web App|
```
Comment (JorTurFer) [inline ADOPTERS.md:10]: Please add your company URL/logo and the message (what the company does?)

Comment (Rohithzr) [reply ADOPTERS.md:10]: Ah! Got it.

Comment (Rohithzr) [reply ADOPTERS.md:10]: | Organization | Status | More Information (Blog post, etc.) |
| ------------ | ---------| ---------------|
|[PropulsionAI](https://propulsionhq.com)|testing|[PropulsionAI](](https://propulsionhq.com)) allows you to add AI to your apps, without writing code.|

Comment (Rohithzr) [reply ADOPTERS.md:10]: @JorTurFer please review the comment. I feel the status (development and testing) is an important measure.

Comment (JorTurFer) [reply ADOPTERS.md]: Yeah, I misunderstood it xD

**Review by JorTurFer — APPROVED** [review]

**Comment by JorTurFer** [comment]
Thanks for sharing that your are using the add-on! ❤️

File: ADOPTERS.md (hunk 1 above)
Comment (Rohithzr) [reply ADOPTERS.md]: Done

**Review by tomkerkhove — APPROVED** [review]
Thanks for using the add-on!

**Pull request merged**